test/parallel/test-dns-perf_hooks.js

```diff
@@ -6,21 +6,55 @@ const dns = require('dns');
 const { PerformanceObserver } = require('perf_hooks');
 
 const entries = [];
-const obs = new PerformanceObserver(common.mustCallAtLeast((items) => {
+const obs = new PerformanceObserver((items) => {
```

Comment:
shouldn't we keep the mustcall function here?

Reply:
Do you mean all the dns operation fail ?